site/_js/firebase.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @fileoverview Firebase integration.
+ */
+import {initializeApp} from 'firebase/app';
+//import { initializeAppCheck, ReCaptchaV3Provider } from "firebase/app-check";
+import {getDatabase, ref, runTransaction, onValue} from 'firebase/database';
+
+export class Firebase {
+  constructor() {
+    this.config = null;
+    // For local Firebase Emulator
+    if (location.hostname === 'localhost') {
+      this.config = {
+        databaseURL: 'http://localhost:9000/?ns=oct-sip-test2',
+      };
+      // this.config = {
+      //   apiKey: "AIzaSyB1zPD5QjfGypRydnHE-F6vWaROlHZebSU",
+      //   authDomain: "oct-sip-test2.firebaseapp.com",
+      //   databaseURL: "https://oct-sip-test2-default-rtdb.firebaseio.com",
+      //   projectId: "oct-sip-test2",
+      //   storageBucket: "oct-sip-test2.appspot.com",
+      //   messagingSenderId: "802581884830",
+      //   appId: "1:802581884830:web:54daed54011bd73be6f08f"
+      // }
+    }
+
+    //Initialize firebase app
+    this.firebase = initializeApp(this.config);
```

Comment:
The return value from `initializeApp()` never seems to be referenced from outside this function, and therefore can be stored in a local variable, instead of an instance variable.

Reply:
This is required in below lines at firecheck and db instance.

Reply:
> On desktop, the years and cookies consent dialogue are overlapping. This would need to be fixed please. ![Screenshot 2022-03-25 6 17 20 PM - Display 2](https://user-images.githubusercontent.com/47141367/160178817-7b02b076-6506-432a-98ae-a9420bd061e2.png)

Done!